nextflow_schema.json

```diff
@@ -552,6 +548,27 @@
                 }
             }
         },
+        "virus_identification_options": {
+            "title": "Virus identification options",
+            "type": "object",
+            "default": "",
+            "properties": {
+                "run_genomad": {
```

Comment:
We were discussing this here (https://github.com/nf-core/mag/issues/455), but for now would it make sense to retain the `skip_{tool}` schematic for consistency?

Reply:
I think that would make sense! Would you like me to go through and update the code to use `skip_genomad` or would you prefer to get feedback from other maintainers first?

Reply:
I think actually for now we should keep it to opt-in (Even if most of the pipeline is opt out). `gunc` checks are opt-in and the original design for mag was rather for bacteria rather than viruses (although that is expanding).

I think it's safe to leave it as it is and we do a overhaul of the parameters to make them more consistent in 3.0